input_output.ipynb

```diff
@@ -1074,9 +1162,15 @@
    },
    "outputs": [],
    "source": [
-    "for i in range(0, 8**3):\n",
-    "    bt = i.to_bytes(4)\n",
-    "    print(f\"bytes: {bt}, integer: {i}, string: {chr(i)}\")"
+    "for i in range(0, 6):\n",
+    "    bt = i.to_bytes(4, 'little')\n",
+    "    print(f\"bytes: {[hex(b) for b in bt]}, integer: {i}, string: {chr(i)}\")\n",
+    "for i in range(60, 74):\n",
+    "    bt = i.to_bytes(4, 'little')\n",
+    "    print(f\"bytes: {[hex(b) for b in bt]}, integer: {i}, string: {chr(i)}\")\n",
+    "for i in range(8**3, 8**3 + 3):\n",
+    "    bt = i.to_bytes(4, 'little')\n",
+    "    print(f\"bytes: {[hex(b) for b in bt]}, integer: {i}, string: {chr(i)}\")"
    ]
   },
   {
```

Comment:
The cell below, under `Converting bytes to text`, when executed give me `TypeError: to_bytes() missing required argument 'byteorder' (pos 2)`

Reply:
Hmm don't think i ever changed that..for consitency i will add the byteorder